Testscripts/Linux/SetupRDMA.sh

```diff
@@ -228,21 +258,26 @@ function Main() {
 		# compile ping_pong
 		cd $ping_pong_help
 		LogMsg "Compiling ping_pong binary in Platform help directory"
-		make -j $(nproc)
+		ret=$(make -j $(nproc))
 		if [ $? -ne 0 ]; then
 			pkey=$(cat /sys/class/infiniband/*/ports/1/pkeys/0)
 			export MPI_IB_PKEY=${pkey}
 			make -j $(nproc)
+			LogMsg "Ping-pong compilation completed"
+		else
```

Comment:
Previous design is then build hit error then add extra steps in if block then make again, otherwise it built successfully, then pass by if block, so here no need else block.

Reply:
Fixed in d560a58210af8358549b783f42f2e5dfa49333ec